app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -0,0 +1,72 @@
+module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::TowerApi
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def create_in_provider(manager_id, params)
+      params = provider_params(params) if respond_to? :provider_params else params
```

Comment:
I don't think this does what you intended.
You're probably looking for something like:
```ruby
params = try(:provider_params, params) || params
```
or
```ruby
params = respond_to?(:provider_params) ? provider_params(params) : params
```